### PR TITLE
IP Policy clarification/revision

### PR DESCRIPTION
**Pull request by ebullient**
Revisions to IP Policy:

- Ensure statements regarding license usage are consistent
- Clarify requirements for contributions
- Require DCO, allow for CLA
- Clarify that non-commercial licenses *can* be used for documentation & samples

### CODE REVIEW

File: policies/ip-policy.md [hunk 1]
```diff
@@ -3,51 +3,77 @@ status: draft
...
-- Seek alternative license approvals from the CFC when necessary.
+CLAs can provide projects with more legal protection, while DCOs offer a simpler, less legally invasive way for developers to contribute.
 
 ## Intellectual Property Rights
 
-CF membership does not grant or transfer intellectual property rights unless explicitly stated by this policy or a Project's license.
+> [!TIP]
+> Joining CF doesn’t grant us any rights to your IP beyond what’s needed for project contributions. Your IP remains yours.
+
+Except for (a) each member’s commitment to be bound by this Policy, the applicable Project Code License(s), and the applicable Project Documentation License(s), in each case with respect to its or its employees’ authorized contributions to any Project; and (b) any applicable contributor license agreement, if any, no license is granted by the member to its intellectual property, and none shall be implied by general membership in the Commonhaus Foundation.
```
Comment (kenfinnigan) [inline policies/ip-policy.md:57]: I'm not entirely sure where (b) ends to indicate what the IP rights are which excludes (a) and (b) situations. I can presume it's after ", if any, ", but I'm not 100% sure. Do we need a semi-colon at the end of (b) like (a) has?

Comment (ebullient) [reply policies/ip-policy.md:57]: you don't usually have a semi-colon at list end. But yea.. that sentence is a mouthful.

An inverted structure could work. Is this better? (I think it covers all the bases..)

> General membership in the Commonhaus Foundation does not imply any license to a member's intellectual property. Members grant no license to their intellectual property to the Commonhaus Foundation, except under the following conditions: (a) their commitment to abide by this Policy, the applicable Project Code License(s), and the applicable Project Documentation License(s) for their and their employees' contributions to any Project; and (b) any applicable contributor license agreement.

Comment (ebullient) [reply policies/ip-policy.md]: Added.

Comment (kenfinnigan) [reply policies/ip-policy.md]: I like this better and don't think anything was dropped

**Review by kenfinnigan — APPROVED** [review]

**Pull request merged**